lib/timescaledb/reporter.ex

```diff
@@ -29,8 +34,9 @@ defmodule Membrane.Telemetry.TimescaleDB.Reporter do
   end
 
   def send_measurement(_) do
```

Comment:
Specs & docs missing

Reply:
Not missing, just not expanded by github.